scripts/xtensa-build-zephyr.py

```diff
@@ -137,6 +137,10 @@ def parse_args():
 						help="Enable debug build")
 	parser.add_argument("-i", "--ipc", required=False, choices=["IPC3", "IPC4"],
 						default="IPC3", help="IPC major version")
+	parser.add_argument("--fw-naming", required=False, choices=["AVS", "SOF"],
+						default="SOF",help="""FW outputs will be named in avs FW style(as dsp_basefw.bin)
+if fw_naming is set to AVS.Otherwise, all firmware files are
```

Comment:
nit pick: missing space AVS. Otherwise

Reply:
let's revise the help information:
```
Firmware naming conversion and folder structure
For SOF:
    /lib/firmware/intel/sof
    └── community
        │   └── sof-tgl.ri
        ├── dbgkey
        │   └── sof-tgl.ri
        └── sof-tgl.ri
For AVS:
    /lib/firmware/intel/avs
    └── tgl
        ├── community
        │   └── dsp_basefw.bin
        ├── dbgkey
        │   └── dsp_basefw.bin
        └── dsp_basefw.bin
```

Reply:
Thanks @aiChaoSONG , will integrate this help into my PR , much clearer and intuitive

Reply:
One more thing to clarify , I don't think for AVS , the parent folder will be named as    /lib/firmware/intel/avs since here we are still generating SOF FW outputs.
The differences just lay in it's under IPC4 mode, subdirectories for different platforms , and also with fw_naming convention of AVS style .
The output will be named as   /lib/firmware/intel/sof-ipc4